pandas_datareader/tiingo.py

```diff
@@ -129,7 +129,7 @@ def read(self):
                 dfs.append(self._read_one_data(self.url, self.params))
             finally:
                 self.close()
-        return pd.concat(dfs, self._concat_axis)
+        return pd.concat(objs = dfs, axis = self._concat_axis)
```

Comment:
Could you please correct the formatting here and in the other line.  Run black if in doubt.